src/spyglass/common/common_lab.py

```diff
@@ -56,8 +56,9 @@ def insert_from_name(cls, full_name):
         """
         labmember_dict = dict()
         labmember_dict['lab_member_name'] = full_name
-        labmember_dict['first_name'] = str.split(full_name)[:-1]
```

Comment:
I know this is a stop-gap, but just to note this, some last names also have spaces in them. For example, my last name can be spelled "De Novellis"

Reply:
Totally agree. We can handle this in a separate PR.